gems/sorbet-runtime/lib/types/types/fixed_hash.rb

```diff
@@ -13,7 +13,15 @@ def initialize(types)
 
     # @override Base
     def name
-      "{#{@types.map {|(k, v)| "#{k}: #{v}"}.join(', ')}}"
+      entries = @types.map do |(k, v)|
+        if Symbol === k
+          "#{k}: #{v}"
```

Comment:
Not sure how far down this rabit hole you care to go:

```
irb(main):001:0> x = :"foo bar"
=> :"foo bar"
irb(main):002:0> "#{x} is it"
=> "foo bar is it"
```

PR is still strictly an improvement, so I'm still fine, but figured I'd call it out.

Reply:
Ohhh, good call. I will go all the way down the rabbit hole :)

Reply:
Ok, I think `":#{k}" == k.inspect" should be guarding those cases too. Pushed the change and added tests for those cases.